package.json

```diff
@@ -22,7 +22,7 @@
     "docs": "run-p docs:*",
     "docs:api": "jsdoc src -r -d docs/api",
     "docs:toc": "doctoc README.md",
-    "lint": "vjsstandard",
+    "lint": "eslint .",
```

Comment:
Why no longer videojs-standard dependency? you can still use .eslintrc files with videojs-standard to override things